__tests__/app.test.js

```diff
@@ -0,0 +1,36 @@
+const request = require("supertest")
+const db = require("../db/connection");
+const app = require("../app");
+const seed = require("../db/seeds/seed");
+const data = require("../db/data/test-data/index");
+
+afterAll(() => {
+  return db.end();
+});
+
+beforeEach(() => {
+  return seed(data);
+});
+
+describe('/api/topics', () => {
+  test('GET: 200 Get all topics', () => {
+    return request(app).get("/api/topics")
+    .expect(200)
+    .then(({body}) => {
+      const {topics} = body;
+
+      expect(Array.isArray(topics)).toBe(true)
```

Comment:
I would argue that this line is made redundant by the next line in your test, iterating through it proves it's an array